src/App.tsx

```diff
@@ -2,10 +2,19 @@ import React from "react"
 
 import './App.css';
 import MovieListPage from "./components/movie-list-page/movie-list-page";
+import { BrowserRouter, Route, Routes } from "react-router-dom";
 
 function App() {
   return (<>
-    <MovieListPage />
+    <BrowserRouter>
+      <Routes>
+        <Route path='/' element={<MovieListPage />}>
+          <Route path=':movieId' element={<MovieListPage />} />
+        </Route>
```

Comment:
this 3 lines can be transformed to one with path='/:movieId'

Reply:
done